Tools/Scripts/generate-gpup-webgl

```diff
@@ -624,21 +624,21 @@ class webkit_ipc_cpp_proxy_impl(object):
         if is_async:
             self.call_stmts += [
                 f"auto sendResult = send(Messages::RemoteGraphicsContextGL::{self.msg_name}({in_exprs}));",
+                "if (sendResult != IPC::ConnectionError::NoError) {",
             ]
         else:
             self.call_stmts += [
                 f"auto sendResult = sendSync(Messages::RemoteGraphicsContextGL::{self.msg_name}({in_exprs}));"
+                "if (!sendResult) {",
```

Comment:
Stuff like this would imply that it'd be good if the send and sendSync would behave the same way wrt the send success (returning truthy value).
So that the programmer doesn't need to think:
- " I send async, I need to check for != IPC::ConnectionError::NoError"
- " I send sync, I need to check for truthy"

Reply:
I added `SendSyncResult::succeeded()` and used that everywhere, which I hope make it easier.